modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/ratings/RatingsManagerImpl.kt

```diff
@@ -26,15 +28,25 @@ class RatingsManagerImpl @Inject constructor(
             .map { it.firstOrNull() ?: noRatings(podcastUuid) }
 
     override suspend fun refreshPodcastRatings(podcastUuid: String, useCache: Boolean) {
-        // The server asks for the ratings to be cached for a period of time. After a user rates ignore the cache to get the new rating.
-        val ratings = cacheServerManager.getPodcastRatings(podcastUuid, useCache)
-        podcastRatingsDao.insert(
-            PodcastRatings(
-                podcastUuid = podcastUuid,
-                total = ratings.total,
-                average = ratings.average,
-            ),
-        )
+        try {
+            // The server asks for the ratings to be cached for a period of time. After a user rates ignore the cache to get the new rating.
+            val ratings = cacheServerManager.getPodcastRatings(podcastUuid, useCache)
+            podcastRatingsDao.insert(
+                PodcastRatings(
+                    podcastUuid = podcastUuid,
+                    total = ratings.total,
+                    average = ratings.average,
+                ),
+            )
+        } catch (e: Exception) {
+            val message = "Failed to refresh podcast ratings"
+            // don't report missing rating or network errors to Sentry
+            if (e is HttpException || e is IOException) {
+                Timber.i(e, message)
+            } else {
+                Timber.e(e, message)
```

Comment:
```suggestion
                LogBuffer.e(LogBuffer.TAG_CRASH, e, message)
```
Maybe this should be the LogBuffer so we can see it in support emails.